templates/base.html

```diff
@@ -91,8 +91,19 @@
     >
       <div class="container">
         <a class="navbar-brand" href="index.html">
-          <img class="logo" src="{{config.logo}}" width="180px" />
+          <img
+             class="logo" src="{{config.logo.image}}"
+             {% if config.logo.height %}
```

Comment:
Probably better to just use https://jinja.palletsprojects.com/en/2.11.x/templates/#if-expression for these sort of expressions.

Reply:
Oh actually this is even better https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#defaulting-undefined-variables

{{ config.image.width | default("") }}

Reply:
:wow: